From 18001c2926dc7d8f57060e34114283d6c2138471 Mon Sep 17 00:00:00 2001
From: Frederik Gladhorn <gladhorn@kde.org>
Date: Thu, 18 Jul 2019 09:55:00 +0200
Subject: [PATCH] Fix build, include QTime

Summary: The type is used, but was not directly included. Fixes build with Qt 5.13.

Reviewers: #plasma, broulik

Reviewed By: #plasma, broulik

Subscribers: plasma-devel

Tags: #plasma

Differential Revision: https://phabricator.kde.org/D22522
---
 kcms/dateandtime/dtime.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/kcms/dateandtime/dtime.h b/kcms/dateandtime/dtime.h
index cceb07145..965b487c6 100644
--- a/kcms/dateandtime/dtime.h
+++ b/kcms/dateandtime/dtime.h
@@ -26,6 +26,7 @@
 
 #include <QSpinBox>
 #include <QComboBox>
+#include <QTime>
 #include <QTimer>
 #include <QWidget>
 #include <QCheckBox>
-- 
2.21.0

